docs/getting-started/custom-ui-ory-elements.mdx

```diff
@@ -69,21 +80,46 @@ Follow these steps to get the sample application code and run it on your machine
    ````mdx-code-block
 
    <Tabs groupId="react-or-preact">
-     <TabItem value="unix" label="Unix systems" default>
-
-   ```shell-session
-   export VITE_ORY_SDK_URL=http://localhost:4000
-   ```
-
-     </TabItem>
-     <TabItem value="powershell" label="Windows Powershell">
-
-   ```powershell
-   $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
-   ```
-
-     </TabItem>
-   </Tabs>
+   <TabItem value="react" label="React" default>
+
+    ```bash
+    # Unix
+    export VITE_ORY_SDK_URL=http://localhost:4000
+    ```
+
+    ```powershell
+    # Windows
+    $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
+    ```
+
+    </TabItem>
+      <TabItem value="preact" label="Preact">
+
+    ```bash
+    # Unix
+    export VITE_ORY_SDK_URL=http://localhost:4000
+    ```
+
+    ```powershell
+    # Windows
+    $Env:VITE_ORY_SDK_URL = "https://{your-project-slug-here}.projects.oryapis.com"
+    ```
+
+    </TabItem>
+      <TabItem value="nextjs" label="Next.js">
+
+    ```bash
+    # Unix
+    export NEXT_PUBLIC_ORY_SDK_URL=http://localhost:4000
+    ```
+
+    ```powershell
+    # Windows
+    $Env:NEXT_PUBLIC_ORY_SDK_URL = "http://localhost:4000"
+    ```
+    [Learn more about environment variables with Next.js](https://nextjs.org/docs/basic-features/environment-variables)
```

Comment:
Put this into an admonition, please. 
https://docusaurus.io/docs/markdown-features/admonitions

Reply:
I've updated the exports to localhost instead, but line 93 is correct.

I'm still trying to get the admonition to work, but it doesn't seem to be detecting the syntax.